packages/sdk/governance/src/proposals.ts

```diff
@@ -144,18 +158,36 @@ export class ProposalBuilder {
    * @param tx A Web3 transaction object to convert.
    * @param params Parameters for how the transaction should be executed.
    */
-  fromWeb3tx = (tx: TransactionObject<any>, params: ProposalTxParams): ProposalTransaction => ({
+  fromWeb3tx = (tx: CeloTxObject<any>, params: ProposalTxParams): ProposalTransaction => ({
     value: params.value,
     to: params.to,
     input: tx.encodeABI(),
   })
 
+  /**
+   * Adds a transaction to set the implementation on a proxy to the given address.
+   * @param contract Celo contract name of the proxy which should have its implementation set.
+   * @param newImplementationAddress Address of the new contract implementation.
+   */
+  addProxyRepointingTx = (contract: CeloContract, newImplementationAddress: string) => {
```

Comment:
this is a relic, can be removed

Reply:
we mention it as an example in celotool/src/cmds/deploy/upgrade/hotfix.ts
Maybe we can remove it in another PR. Don't want to keep adding things to this.